mern/client/src/screens/SavingSession/savingSession.js

```diff
@@ -52,6 +80,33 @@ export default function SavingSession() {
       );
   };
 
+  const resetEnd = () => {
+    setSessionID(null);
+    setcurrQAState({});
+    setFlowState({});
+    setFlowStartingQuestions([]);
+    setFlowAllQuestions([]);
+    setSpeaker("User:");
+    setPrevSpeaker("Bot:");
+    setQuestionState([]);
+    setNextQuestions([]);
+    setAllQuestions([]);
```

Comment:
Again maybe make a helper function for this. Maybe make a new file just for resetting this since it happens twice? A function that just imports the contexts, with a singular function that resets them.